data_ns: ''
maneuver_ns: ''
visual_ns: ''

tf_broadcaster:
  process_freq: 50.0
  
  # flag
  is_finding_home: false

  exp_camera_height_m: 1.5
  
  extrinsic_imu_to_camera_x: 0.1257
  extrinsic_imu_to_camera_y: 0.0
  extrinsic_imu_to_camera_z: 0.0
  
mission_manager:
  process_freq: 30.0

usb_cam:
  camera_frame_id: 'camera_link'